nexus/src/app/background/tasks/blueprint_rendezvous.rs

```diff
@@ -92,7 +92,11 @@ impl BlueprintRendezvous {
 
         // Return the result as a `serde_json::Value`
         match result {
-            Ok(()) => json!({}),
+            Ok(stats) => json!({
```

Comment:
non-blocking suggestion: I'd add a struct in `nexus/types/src/internal_api/background.rs` that can be shared with omdb instead of free form json here.

Reply:
Thanks; fixed in [5410b4b](https://github.com/oxidecomputer/omicron/pull/7397/commits/5410b4b437fe89e8b1941edf99dce9428017ad27)